frame/evm/src/lib.rs

```diff
@@ -194,13 +217,14 @@ decl_module! {
 			Ok(())
 		}
 
-		#[weight = WeightForCallCreate::<T::FeeCalculator>::default()]
-		fn call(origin, target: H160, input: Vec<u8>, value: U256, gas_limit: u32)
-			-> dispatch::Result
-		{
+		#[weight = WeightForCallCreate]
+		fn call(
+			origin, target: H160, input: Vec<u8>, value: U256, gas_limit: u32, gas_price: U256,
```

Comment:
style nit: it's `(long, long, long)` or `(
    high,
    high,
    high
)`. not a combination.